static/app/components/searchSyntax/parser.tsx

```diff
@@ -315,21 +351,20 @@ class TokenConverter {
     });
 
   tokenFilter = <T extends FilterType>(
-    type: T,
+    filter: T,
     key: FilterMap[T]['key'],
     value: FilterMap[T]['value'],
     operator: FilterMap[T]['operator'] | undefined,
     negated: FilterMap[T]['negated']
   ) =>
     this.makeToken({
       type: Token.Filter,
-      filter: type,
-      config: filterTypeConfig[type],
```

Comment:
The `config` object has been removed.

This should now likely be looked up from `const types = token.invalid?.expectedType ?? [token.filter]` (or something similar)